storage/fuzz/fuzz_targets/adb_operations.rs

```diff
@@ -4,7 +4,7 @@ use arbitrary::Arbitrary;
 use commonware_cryptography::Sha256;
 use commonware_runtime::{buffer::PoolRef, deterministic, Runner};
 use commonware_storage::{
-    adb::any::{Any, Config},
+    adb::any::fixed::{Any, Config},
```

Comment:
I took a similar approach when I split `Archive` (only testing `prunable`)...we should probably create issues to also add fuzzing for variable (and for `immutable`).